src/App.tsx

```diff
@@ -15,71 +16,81 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export class App extends Component {
+  state = {
+    selectedGood: 'Jam',
+  };
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  handleAddClick = (good: string) => {
```

Comment:
```suggestion
  handleSelectGood = (good: string) => {
```

Use a proper name that describes an action, not the reason